sparkles/tests/test_find_er_catalog.py

```diff
@@ -0,0 +1,145 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import warnings
+from proseco import get_aca_catalog
+from proseco.tests.test_common import mod_std_info
+import numpy as np
+from Quaternion import Quat
+import Ska.Sun
+
+from sparkles.find_er_catalog import (
+    get_candidate_stars, find_er_catalog, filter_candidate_stars_on_ccd,
+    get_guide_counts, init_quat_from_attitude)
+
+
+# Known tough field: PKS 0023-26 pointing
+ATT = Quat([0.20668099834, 0.23164729391, 0.002658888173, 0.9505868852])
+DATE = '2021-09-13'
+T_CCD = -8.0
+
+# Get initial catalog at the PKS 0023-26 attitude. Ignore the penalty limit for
+# this work.
```

Comment:
Why ignore the penalty limit?

Reply:
(I suppose I just figure it is unlikely we'll set the penalty limit colder, so it is going to be pretty moot anyway)